app/managing-users.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: page
+title: Managing users
+analytics_key: managing-users
+parent:
+  title: How to use the pilot service
+  url: /guide
+previous:
+  title: Reporting
+  url: /reporting
+next:
+  title: Managing vaccines
+  url: /managing-vaccines
+---
+
+{% include 'guide-contents.njk' %}
+
+## Adding new users  
+
+Only lead administrators can add and manage users. 
+
+To add a user, you must enter their: 
+
+* First name  
+* Last name 
+* NHS-approved email address  
+* Clinician status
+
+## Selecting clinician status 
+
+Select ‘yes’ if users are registered clinicians. This will allow them to assess patients and record their consent.  
+
+## Selecting permission levels 
+
+Choose and select a permission level for new users: 
+
+| **Permission level**               | **Lead administrator**    | **Administrator**    | **Recorder**    |
+|------------------------------------|---------------------------|----------------------|-----------------|
+| **Records and edits vaccinations** | Yes                       | Yes                  | Yes             |
+| **Adds and manages vaccines**      | Yes                       | Yes                  | No              |
+| **Creates reports**                | Yes                       | Yes                  | No              |
+| **Adds and manages users**         | Yes                       | No                   | No              |
+
+You can select more than one user for each permission level. 
+
+![Screen shows user adding a new user's details and selecting their permission level](/images/manage-users-add.png)
+
+## Creating a RAVS account  
+
+Once you have added a new user, we’ll automatically create a RAVS and Okta user account. We'll also send them a RAVS welcome email with instructions to access the service. Okta allows new users to verify their identity and use RAVS.  
```

Comment:
This sentence is slightly contradicted by the one below about using an existing Okta account if they have one. Maybe shorten it and leave the description of setting up the Okta account to the 2 bullets below?

Reply:
Thanks for the spot, agree with that so how about:

Once you have added a new user, we’ll automatically create a RAVS account and send them a welcome email to access the service.  

Okta allows new users to verify their identity and use RAVS.  

- If users already have an Okta account for other NHS applications, then their RAVS account will be added to their existing Okta account. 
- If users are new to Okta, they’ll also receive an Okta email with instructions to set up their account. They’ll need to do this first to access RAVS.